### ISSUE TITLE
Loaders should be configurable

### ISSUE THREAD

**Issue by kirill-konshin** [insin/nwb#303]
This issue is a: Bug report.

- [x] Bug report
- [ ] Feature request
- [ ] Question / support request
- [ ] Other

As mentioned in #294 providing `loader` in config leads to unexpected results:

```
{
  style: {
    loader: require.resolve('css-loader/useable'),
    options: {modules: true}
  }
}
```

Produces:

```
[ { loader: '/boilerplate/node_modules/style-loader/index.js',
    options: 
     { loader: '/boilerplate/node_modules/style-loader/useable.js', modules: true } },
```

Following two configs produce same results:

```
css: {options: {modules: true}}
css: {modules: true}
```

**Comment by kirill-konshin** [comment]
Is there any progress? We are considering to move our React stack to NWB and this issue is a roadblock...

**Comment by insin** [comment]
`modules` (and other options) being moved into an `options` object is intended so you don't have to have another level of nesting in your config to tweak loader options.

`loader` was incorrectly being moved into the `options` object - this has been fixed in the `next` branch.

**Comment by kirill-konshin** [comment]
How to install it? I tried `npm i nwb@next` and it did not work... Since your sources are compiled to `lib` dir I can't just install GitHub branch, or can I?

What would be the correct config? This one should work?

```
{
  style: {
    loader: require.resolve('css-loader/useable'),
    options: {modules: true}
  }
}
```

**Comment by kirill-konshin** [comment]
Any plans to release a new NPM version?

**Comment by insin** [comment]
See #310

**Comment by kirill-konshin** [comment]
What's the ETA?

**Comment by insin** [comment]
Don't have the time for an ETA 😼

**Comment by kirill-konshin** [comment]
Sad... okay, as I said, we're waiting for specific fix of at least loaders... can you release 0.16-beta.1 for example? )

**Comment by insin** [comment]
You could publish your own version of the next branch temporarily:

```
git clone https://github.com/insin/nwb.git
cd nwb
git checkout next
npm install
npm run build
# set package.json name to @yournpmusername/nwb and version to whatever you want
npm publish --access public
```

I'd do it but I have limited time for open source and am in the middle of something I want to get finished while I have time today.

**Comment by kirill-konshin** [comment]
Got it, thanks.

**Comment by insin** [comment]
This should be fixed in v0.16.0

**Comment by kirill-konshin** [comment]
Confirmed.
